public/templates/newJob.dust

```diff
@@ -7,24 +7,20 @@ Matador
 {<body}
 <h1 class="page-header">New  Job</h1>
 
-    <form role="form" id="form1" name="form1" method="post" action="{basename}/api/jobs/create" enctype="application/json">
-    <div class="form-group">
-      <label for="queue">Queue:</label>
-      <br>
-      <input name="queue" id="queue" class="form-control" {?error}value="{queue}"{/error}></input>
-      <br>
-      <label>Payload:</label>
-      <br>
-      <textarea name="payload" id="textarea"  class="form-control" cols="45" rows="5" >{?error}{payload}{/error}</textarea>
-      <br>
-      <input type="submit" name="button" id="button" class="btn btn-default" value="Create Job" />
-    </div>
+    <form role="form" id="newjob">
+      <div class="form-group">
+        <label for="queue">Queue:</label>
+        <br>
+        <input name="queue" id="queue" class="form-control"></input>
+        <br>
+        <label for="payload">Payload:</label>
+        <br>
+        <textarea name="payload" id="textarea"  class="form-control" cols="45" rows="5" ></textarea>
+        <br>
+        <input onclick="redisHandler.fn.createJob()" id="button" class="btn btn-default" value="Create" />
+      </div>
     </form>
 
-    {?error}
-    <div class="alert alert-danger">
-    <strong>Error!</strong> {error}
-    </div>
-    {/error}
+    <div class="alert alert-danger hidden"></div>
```

Comment:
Does removing the `{?error}` block mean that errors wont be displayed if there is an error with the `GET` request?